db/arena_wrapped_db_iter.cc

```diff
@@ -81,7 +81,17 @@ Status ArenaWrappedDBIter::Refresh() {
         latest_seq, /* allow_unprepared_value */ true);
     SetIterUnderDBIter(internal_iter);
   } else {
-    db_iter_->set_sequence(db_impl_->GetLatestSequenceNumber());
+    SequenceNumber latest_seq = db_impl_->GetLatestSequenceNumber();
+    // Refresh range-tombstones in MemTable
+    if (!read_options_.ignore_range_deletions) {
+      SuperVersion* sv = cfd_->GetReferencedSuperVersion(db_impl_);
```

Comment:
This reference is never `Unref()`ed. I'm not sure it's safe to use either because the `SuperVersion` might have changed after we compared against `cur_sv_number`.

This Refresh() fast-path is tricky in the presence of range tombstones; I will think about it more.

Reply:
Thanks for your review.
Yes, this way does fix the above problem, but it may be a little tricky, and I'm not sure how to solve it perfectly. 
And I found that it got an error when closing rocksdb, it may be caused by the reference is never Unref()ed? I can try to fix this point.